ios/RNModules/RNLocalSearch/RNLocalSearch/RNLocalSearch.m

```diff
@@ -0,0 +1,94 @@
+#import <MapKit/MapKit.h>
+#import "RNLocalSearch.h"
+#import <React/RCTConvert.h>
+#import <CoreLocation/CoreLocation.h>
+#import <React/RCTConvert+CoreLocation.h>
+#import <React/RCTUtils.h>
+
+@interface RCTConvert (Mapkit)
+
++ (MKCoordinateSpan)MKCoordinateSpan:(id)json;
++ (MKCoordinateRegion)MKCoordinateRegion:(id)json;
+
+@end
+
+@implementation RCTConvert(MapKit)
+
++ (MKCoordinateSpan)MKCoordinateSpan:(id)json
+{
+    json = [self NSDictionary:json];
+    return (MKCoordinateSpan){
+        [self CLLocationDegrees:json[@"latitudeDelta"]],
+        [self CLLocationDegrees:json[@"longitudeDelta"]]
+    };
+}
+
++ (MKCoordinateRegion)MKCoordinateRegion:(id)json
+{
+    return (MKCoordinateRegion){
+        [self CLLocationCoordinate2D:json],
+        [self MKCoordinateSpan:json]
+    };
+}
+
+@end
+
+@implementation RNLocalSearch
+{
+    MKLocalSearch *localSearch;
+}
+
+- (dispatch_queue_t)methodQueue
+{
+    return dispatch_get_main_queue();
+}
+
+RCT_EXPORT_MODULE()
+
+#pragma mark -
+#pragma mark format RCT callback response
```

Comment:
You don't need pragma mark - it's needed (but optional) to mark section in the class. In your case you just exporting module with the method - pragma mark is redundant.